example/server/integrations.py

```diff
@@ -400,13 +464,16 @@ def put(self, params: Dict):
         if not all(val in params for val in required_fields):
             raise ValueError(f"required fields: {', '.join(required_fields)}")
 
+        # query params for fetching/creating the PolarisStellarAccount
+        qparams = {"account": params["account"]}
+        if "memo" in params:
+            qparams["memo"] = params["memo"]
+            qparams["memo_type"] = params["memo_type"]
```

Comment:
Possible to pass a memo with no memotype?

Reply:
Nope, `memo_str` will raise an error in that case